decidim-elections/lib/decidim/elections/admin_engine.rb

```diff
@@ -30,6 +30,10 @@ class AdminEngine < ::Rails::Engine
                 get :proposals_picker
                 resource :proposals_import, only: [:new, :create]
               end
+              member do
+                get :select
+                get :unselect
```

Comment:
Since those methods changes the database state it's better to use the `PUT` http method. You will need to change your links to emulate this behavior as well.